documentation/SDKs/javascript/README.md

```diff
@@ -0,0 +1,358 @@
+```
+#### Level Drop Offs - monthly
+You can use the ***Nether.Analytics.GetLevelDropOffsMonthly*** method to get the total count of level drop offs by level and month. Returns an arry of DropOff
+##### Example usage
+```javascript
+var dropOffs = Nether.Analytics.GetLevelDropOffsMonthly();
+console.log('Level ' + dropOffs[0].ReachedLevel + 'Drop offs: ' + dropOffs[0].TotalCount + ' for ' + dropOffs[0].Month + '-' + dropOffs[0].Year);
+```
+#### Level Drop Offs - yearly
+You can use the ***Nether.Analytics.GetLevelDropOffsYearly*** method to get the total count of level drop offs by level and year. Returns an arry of DropOff
+##### Example usage
+```javascript
+var dropOffs = Nether.Analytics.GetLevelDropOffsYearly();
+console.log('Level ' + dropOffs[0].ReachedLevel + 'Drop offs: ' + dropOffs[0].TotalCount + ' for ' + dropOffs[0].Year);
+```
+### Identiy
+
+### Leaderboard
+
+#### Get Leaderboard
+You can use the ***Nether.Leaderboard.GetLeaderboard*** method to get the currently configured Leaderboard. Returns an array of Leaderboard
+##### Example usage
+```javascript
+var leaderboard = Nether.Leaderboard.GetLeaderboard();
+```
+### Get Leaderboard by name
+You can use the ***Nether.Leaderboard.GetLeaderboardFromName*** method to get all of the gammer tags and highscores for the specified Leaderboard. Returns a LeaderboardSummary object
+#### Example usage
+```javascript
+var leaderboardScores = Nether.Leaderboard.GetLeaderboard('testboard');
+```
+
+### Put score
+You can use the ***Nether.Score.PutScore*** method to add a players score.
+#### Example usage
+```javascript
+Nether.Scores.AddScore('UK', 100, 'customTag');
+Nether.Scores.AddScore('UK', 100);
+```
+
+### Drop scores
+You can use the ***Nether.Score.DropMyScore*** method to delete all scores for the logged in player
+#### Example usage
+```javascript
+Nether.Scores.DropMyScore();
+```
+### Player Management
+
+### Get the current Player
+You can use the ***Nether.Player.GetCurrentPlayer*** method to get the current player details. Returns a Player object
+#### Example usage
+```javascript
+var player = Nether.Player.GetCurrentPlayer();
+```
+### Put Player
+You can use the ***Nether.Player.PutCurrentPlayer*** method to update Players details
+#### Example usage
+```javascript
+Nether.Player.PutCurrentPlayer('gammerTag', 'country', 'customTag');
+```
+### Add current player to group
+You can use the ***Nether.Player.AddCurrentPlayerToGroup*** method to add the current player to a group
+#### Example usage
+```javascript
+Nether.Player.AddCurrentPlayerToGroup('groupName');
+```
+
+### Delete current player
+You can use the ***Nether.Player.DeleteCurrentPlayer*** method to delete the current player
+#### Example usage
+```javascript
+Nether.Player.DeleteCurrentPlayer();
+```
+
+### Get current player state
+You can use the ***Nether.Player.GetCurrentPlayerState*** method to get the state of the current player. Returns a PlayerState object
```

Comment:
I think we should revisit player state management later on. For now I would suggest that we player state is a string, that could be JSON, but not necessary.